src/components/publishing/embed.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import styled, { StyledFunction } from "styled-components"
+
+interface EmbedProps {
+  section: any
+}
+
+interface FrameProps extends React.HTMLProps<HTMLIFrameElement> {
+  mobileHeight?: number
+  height: number
+}
+
+const iframe: StyledFunction<FrameProps & React.HTMLProps<HTMLIFrameElement>> = styled.iframe
+
+const Embed: React.SFC<EmbedProps> = props => {
+  const { url, height, mobile_height } = props.section
+  return <IFrame src={url} scrolling="no" frameBorder="0" height={height} mobileHeight={mobile_height} />
+}
+
+const IFrame = iframe`
+  width: 100%;
+  height: ${props => props.height + "px"};
+  @media (max-width: 600px) {
+    height: ${props => props.mobileHeight + "px"};
+  }
```

Comment:
@alloy maybe you have some insight here -- ideally this will look like
```
${pMedia.sm`
   height: ${props => props.mobileHeight + "px"}
`}
```
but I keep getting 
```
Property 'mobileHeight' does not exist on type 'ThemeProps<any>'
```
I've been messing around with the media query helper to try unpack what's happening but I'm pretty lost as to why TS complains when this just wraps some CSS and the ThemeProps accepts an `<any>`. Any leads would help!